doc/admin.md

```diff
@@ -25,6 +25,24 @@ You can use the php-mail function to send mails. This is needed for some webhost
 ```php
 'app.mail.transport' => 'php-mail'
 ```
+### Provisioning
+to Provisioning of Mail Accounts for all user navigate to Settings -> Groupware
```

Comment:
I'm fine with this line, but the rest should be on the UI. Docs here are prone to get out of sync with the UI changes.